opengrok-indexer/src/main/java/org/opengrok/indexer/web/messages/Message.java

```diff
@@ -161,7 +171,7 @@ public void serialize(
                 final SerializerProvider serializerProvider
         ) throws IOException {
             if (duration != null) {
-                jsonGenerator.writeString(duration.toString());
+                jsonGenerator.writeString(Util.encode(duration.toString()));
```

Comment:
is it necessary to encode duration iso standard?

Reply:
probably not. Does not hurt to be extra careful, though. Again, this is in line with original encoding code.